node/src/main/scala/coop/rchain/node/Main.scala

```diff
@@ -28,6 +29,19 @@ object Main {
     implicit val deployService: DeployService[Task] =
       new GrpcDeployService(conf.grpcHost(), conf.grpcPort())
 
+    val certificate = conf.certificate.map(c => Try(CertificateHelper.from(c))).toOption match {
```

Comment:
This seem to be part of `NodeRuntime` - that should know how to run itself (including checking for certificates). `Main` should just run the runtimes, nothing else - should be as thin as possible.